code/modules/hydroponics/hydro_tray_process.dm

```diff
@@ -235,6 +235,12 @@
 				continue
 			environment.adjust_gas(gas, -(seed.consume_gasses[gas]), FALSE)
 		environment.update_values()
+		
+	// Special case for plasma cabbage plants to absorb plasma from the air
+	// doesn't affect seed/product potency because i can't figure out how to use check_for_divergence()
+	if(istype(seed,/datum/seed/plasmacabbage) && environment[GAS_PLASMA] > 0)
+		environment.adjust_gas(GAS_PLASMA, min(-2,environment[GAS_PLASMA]), FALSE)
```

Comment:
why the `min()`?
it'll always be -2, won't it?

Reply:
its supposed to be min(-2, -environment[GAS_PLASMA]) so the plasma doesn't go negative (it happened during my tests)